### PR TITLE
[UXE-3328] bug: profile menu position

### PR DESCRIPTION
**Pull request by cesaroeduardo**
### Explain what was fixed and the correct behavior.
The account menu gets lost in the anchoring and appears incorrectly after scrolling the screen, a vertical positioning value was added next to the style so that it stays fixed at that point on the screen, I tested via passthrough and class and it didn't work. returned the expected behavior, just via style.

### Does this PR introduce UI changes? Add a video or screenshots here.

https://github.com/aziontech/azion-console-kit/assets/44036260/c5c1cb57-71ec-40ac-9155-2e792823a537

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] bug: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [x] Application responsiveness was tested to different screen sizes
- [ ] New tests are added to prevent the same issue from happening again
- [x] UI changes are validated by a team designer
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [x] Edge
- [x] Firefox
- [ ] Safari

### CODE REVIEW

File: src/templates/profile-block/index.vue [hunk 1]
```diff
@@ -109,12 +109,12 @@
   </Sidebar>
 
   <PrimeMenu
-    style="position: fixed"
+    style="position: fixed !important; top: 46px"
```
Comment (matheus-sunderhus) [inline src/templates/profile-block/index.vue:112]: não seria possível passar isso via :pt usando classes tailwind (fixed top-12)?
root: { class: 'w-[280px] pb-2 pt-0' },
root: { class: 'w-[280px] pb-2 pt-0 fixed top-12' },

**Review by matheus-sunderhus — CHANGES_REQUESTED** [review]
Verificar uso de classes tailwind e pt props, para evitar uso de estilo fora do tailwind.

File: src/templates/profile-block/index.vue (hunk 1 above)
Comment (cesaroeduardo) [reply src/templates/profile-block/index.vue:112]: Como coloquei ali na descrição, foi a primeira alternativa, por algum motivo colocando no passtrought ao clicar ele sobe o scroll que é uma interação meio estranha, colocando via style não teve esse problema, por isso mantive.

Comment (cesaroeduardo) [reply src/templates/profile-block/index.vue:112]: https://github.com/aziontech/azion-console-kit/assets/44036260/cbadd42d-08bf-40c7-a554-c82d58ebe2cf

Comment (matheus-sunderhus) [reply src/templates/profile-block/index.vue:112]: enconrei esse ajuste aqui:
image

colocando isso no código consegui manter o o menu junto do header.

**Review by matheus-sunderhus — APPROVED** [review]

**Pull request merged**